corehq/apps/accounting/management/commands/get_minimum_features_by_domain.py

```diff
@@ -22,7 +23,7 @@ def add_arguments(self, parser):
     def handle(self, domain_names, **kwargs):
         privileges = sorted(_privilege_to_response_function().keys())
         print(','.join(['Project Space'] + privileges + ['Lowest Plan']))
-        for domain in filter(lambda domain: domain, map(Domain.get_by_name, domain_names)):
+        for domain in [domain for domain in map(Domain.get_by_name, domain_names) if domain]:
```

Comment:
You can also do `filter(None, [...])` to remove falsy values from a list.  This is fine too, though.